tests/tls_challenge_test.go

```diff
@@ -74,6 +75,14 @@ func (ts *TLSChallengeSuite) TearDownSuite() {
 }
 
 func (ts *TLSChallengeSuite) TestTLSChallenge() {
+	// When COLLECTION_METHOD is set to NO_COLLECTION the collector DS contains only a compliance container but
+	// no collector container, and this test will trigger the ClusterHealth updater which then queries the collectorDS
+	// for a container that does not exist, sending Sensor into an Error state.
+	// See sensor/kubernetes/clusterhealth/updater.go:getCollectorInfo()
```

Comment:
This is a very long sentence that may be challenging to read. Here is my attempt at rephrasing it:
```suggestion
	// This test relies on several log lines appearing in the Sensor logs. One of those does not appear
	// when running in 3.74. This is caused by Collector being set to NO_COLLECTION method
	// which results in no Collector container in the pod (only Compliance is present).
	// That condition makes it impossible for Sensor to parse the Collector image version from 
	// the Collector container because such container is absent. This sends Sensor into an error state
	// and the said log line is never produced. A fix for that is trivial, but would require a patch release for 3.74 
	// and we do not do patch releases for this version anymore.
	// See getCollectorInfo() in sensor/kubernetes/clusterhealth/updater.go for implementation details
```

Reply:
Your version is easier to read, I'll take it. Thanks